web/src/ui/commons.js

```diff
@@ -0,0 +1,50 @@
+const Select = ({
+    label,
+    options,
+    onSelect,
+    disabled = false,
+    className = ""
+}) => {
+    return (
+        <div className="w-full mx-auto">
+            <select
+                id={label}
+                className={className}
+                disabled={disabled}
+                onChange={(e) => onSelect(e.target.value)}
+            >
+                <option defaultValue>Select {label}</option>
```

Comment:
Cool, kannte ich noch garnicht...ist offenbar erst in einer neuen Version dazugekommen.